Packs/FireMonSecurityManager/Integrations/FireMonSecurityManager/FireMonSecurityManager.py

```diff
@@ -0,0 +1,1078 @@
+                                          )
+        return api_response
+
+    def validate_pca_change(self, payload_pca, pca_url_suffix, headers):
+        api_response = self._http_request(method='POST',
+                                          url_suffix=pca_url_suffix,
+                                          json_data=payload_pca,
+                                          headers=headers,
+                                          params=None,
+                                          timeout=20)
+        return api_response
+
+    def rule_rec_api(self, auth_token, payload):
+        """ Calling orchestration rulerec api by passing json data as request body, headers, params and domainId
+                which returns you list of rule recommendations for given input as response"""
+
+        parameters = {'deviceGroupId': payload["deviceGroupId"], 'addressMatchingStrategy': 'INTERSECTS',
+                      'modifyBehavior': 'MODIFY', 'strategy': None}
+        data = get_rule_rec_request_payload()
+
+        data["destinations"] = payload["destinations"]
+        data["sources"] = payload["sources"]
+        data["services"] = payload["services"]
+        data["action"] = payload["action"]
+        rule_rec_api_response = self._http_request(
+            method='POST',
+            url_suffix=RULE_REC_URL.format(payload["domainId"]),
+            json_data=data,
+            params=parameters,
+            headers={
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+                'X-FM-Auth-Token': auth_token,
+            }
+        )
+        return rule_rec_api_response
+
+    def rule_rec_output(self, auth_token, payload):
+        """ Calling orchestration rulerec api by passing json data as request body, headers, params and domainId
+                which returns you list of rule recommendations for given input as response"""
+
+        parameters = {'deviceId': payload["deviceId"], 'addressMatchingStrategy': 'INTERSECTS',
+                      'modifyBehavior': 'MODIFY', 'strategy': None}
+        data = get_rule_rec_request_payload()
+
+        data["destinations"] = payload["destinations"]
+        data["sources"] = payload["sources"]
+        data["services"] = payload["services"]
+        data["action"] = payload["action"]
+        rule_rec_api_response = self._http_request(
+            method='POST',
+            url_suffix=RULE_REC_URL.format(payload["domainId"]),
+            json_data=data,
+            params=parameters,
+            headers={
+                'Content-Type': 'application/json',
+                'Accept': 'application/json',
+                'X-FM-Auth-Token': auth_token,
+            }
+        )
+        return rule_rec_api_response
+
+
+def test_module(client):
+    """
+     Performs basic get request to get item samples
+     """
+    response = client.authenticate_user()
+    if response.get('authorized'):
+        demisto.results("ok")
```

Comment:
You will use the return_result on the main function, here you just need to return a string "ok"

Reply:
resolving as outdated.